src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -155,13 +156,27 @@ export const ChooseApiPage = () => {
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.give_access_to_new_api')}</PageHeader>
         <PageContent>
           {chosenDelegableOrgs.length < 1 ? (
-            <Panel
-              title={t('common.error')}
-              variant={PanelVariant.Warning}
-              forceMobileLayout={isSm}
-            >
-              {t('api_delegation.orgs_not_chosen_subtitle')}
-            </Panel>
+            <RestartPrompter
+              spacingBottom
+              button={
+                <Button
+                  variant='secondary'
+                  color='danger'
+                  onClick={() => {
+                    navigate(
+                      '/' +
+                        ApiDelegationPath.OfferedApiDelegations +
+                        '/' +
+                        ApiDelegationPath.ChooseOrg,
+                    );
+                  }}
+                >
+                  {t('common.restart')}
+                </Button>
+              }
+              title={t('common.an_error_has_occured')}
+              ingress={t('api_delegation.delegations_not_registered')}
```

Comment:
I can change it to what you suggest but I think a browser alert on refresh is unneccessary when we have this alert

Reply:
And are you thinking color change on Alert on just this page or changing all alerts to yellow?

Reply:
We should probably use the same color on all of them I think.
Maybe with just the regular colored button? Since there isn't really any danger associated with the action of starting over.

But I think we just have to try out a few versions and see what works the best.

Reply:
Maybe like this?
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/ac91d649-d52a-4939-a8df-ec63f0980e88)

Reply:
(Though I keep coming back to the feeling that it might just be simpler and more user friendly to just redirect the user to the start page automatically when there's nothing for them to do on the page anyway 😅)

Reply:
Redirect user to the start page automatically will make it impossible for us to debug if we have a problem in redux where Redux doesn't store orgs, apis or services. Then user will be redirected to start page and don't know why he is sent back. He will send us a screen shot of just the start page.

With that being said it might be better if button in the Alert takes them to start page. But I'm not sure if they'd be confused when they have to start the process whole process all over again from the start page

Reply:
Not impossible, but trickier yeah.
But it will also likely be a better experience for the user, I would think.

But this would be it's own separate issue anyway.

Also, yeah, I think I would expect the button to take me back to the overview page. Especially if I got to the page through a link or bookmark, which I think are the most likely scenarios.

Reply:
I think this is a UX task to test where we should send users, so we can probably talk to them about this if we get many problems with it